src/client/pfs/pfs.proto

```diff
@@ -76,6 +76,8 @@ message Commit {
 // CommitInfo is the main data structure representing a commit in etcd
 message CommitInfo {
   Commit commit = 1;
+  // User-provided message describing the contents of the commit
+  string commit_message = 8;
```

Comment:
Could we drop `commit` here, I think it's redundant since the proto message is already a `CommitInfo` and could we use `description` instead of `message` since that's what the equivalent field on `repos` and `pipelines` is called. Git does call it a commit message so I see the argument but I think internal consistency is more important than consistency with Git.

Reply:
Done, though I left the flag on `finish-commit` as `--message` and `-m` (to match Git). Let me know if you think I should change those as well

Reply:
I think having `--message` and `-m` work with `finish-commit` is nice ergonomics. I think having `--description` as a synonym is probably good too. Also can we have these flags on `start-commit` and `finish-commit`? I feel like you may know what your doing when you start the commit, then you add a bunch of stuff and you come back and are like "wait, what did I just add?"

Reply:
Done. Per our discussion:
- `--message` and `--description` are now both flags on `start-commit` and `finish-commit`
- Those flags are synonyms of each other